src/pyOpenMS/pyopenms/_dataframes.py

```
@@ -657,77 +706,41 @@ def _add_meta_values(df: _pd.DataFrame, object: any) -> _pd.DataFrame:
-MSSpectrum.__module__ = _MSSpectrum.__module__
-MSSpectrum.__name__ = 'MSSpectrum'
-
-PeakSpectrum = _MSSpectrumDF
-PeakSpectrum.__module__ = _PeakSpectrum.__module__
-PeakSpectrum.__name__ = 'PeakSpectrum'
+def to_pandas(data_dict):
+    """Convert a dictionary of NumPy arrays to a pandas DataFrame if pandas is installed."""
+    try:
+        import pandas as pd
+    except ImportError:
+        raise ImportError("The 'pandas' package is required to use this function.")
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)